web/components/roadmap/MiniRoadmap.test.tsx

```diff
@@ -28,31 +38,26 @@ describe("<MiniRoadmap />", () => {
   });
 
   it("expands the step that you are in by default", () => {
-    useMockUserData({});
```

Comment:
why switch everything to using stateful? do you think we should do that more generically? I always assumed it felt kinda like overkill for a lot of these tests to render statefully when the test contents don't care about that. but I could be wrong